pom.xml

```diff
@@ -270,6 +270,29 @@
                 <javadoc.doclint.none>-Xdoclint:none</javadoc.doclint.none>
             </properties>
         </profile>
+        <profile>
+            <id>dev</id>
+            <build>
```

Comment:
I'd prefer simply setting Maven properties (i. e. [`maven.javadoc.skip`](https://maven.apache.org/plugins/maven-javadoc-plugin/javadoc-mojo.html#skip) and [`findbugs.skip`](https://gleclaire.github.io/findbugs-maven-plugin/check-mojo.html#skip)) so that these can still be overridden on the command line. Running `mvn -Pdev -Dfindbugs.skip=false` would for example skip the Javadoc generation but still run the FindBugs checks.

This change would also make the profile itself shorter:
```xml
<profile>
    <id>dev</id>
    <properties>
        <maven.javadoc.skip>true</maven.javadoc.skip>
        <findbugs.skip>true</findbugs.skip>
    </properties>
</profile>
```